lib/Factory/WidgetFactory.php

```diff
@@ -266,6 +266,7 @@ public function query($sortOrder = null, $filterBy = [])
               widget.type,
               widget.duration,
               widget.displayOrder,
+              widget.schemaVersion,
```

Comment:
I think we're going to need `schemaVersion` in the `intProperties` when we hydrate?